Runtime/DisguiseRenderStream.cs

```diff
@@ -542,9 +554,8 @@ public IEnumerable<RenderTexture> InputTextures
         public FrameData LatestFrameData { get; protected set; }
 
         public bool HasNewFrameData { get; protected set; }
-        
-        GameObject[] m_Cameras = { };
-        ManagedSchema m_Schema = new ();
+
```

Comment:
Minor / Preference: I'm not a big fan of tuple because it creates super long typenames that need to be fixed a little bit everywhere in the code whenever we add / remove something in it.  This is why I instead prefer a simple struct...